tests/test_frame.py

```diff
@@ -1821,24 +1822,30 @@ def test_frame_getitem_isin() -> None:
 def test_to_excel() -> None:
     df = pd.DataFrame(data={"col1": [1, 2], "col2": [3, 4]})
 
-    with ensure_clean() as path:
-        df.to_excel(path, engine="openpyxl")
-        check(assert_type(pd.read_excel(path), pd.DataFrame), pd.DataFrame)
-    with ensure_clean() as path:
-        df.to_excel(Path(path), engine="openpyxl")
-        check(assert_type(pd.read_excel(path), pd.DataFrame), pd.DataFrame)
-    with ensure_clean() as path:
-        df.to_excel(path, engine="openpyxl", startrow=1, startcol=1, header=False)
-        check(assert_type(pd.read_excel(path), pd.DataFrame), pd.DataFrame)
-    with ensure_clean() as path:
-        df.to_excel(path, engine="openpyxl", sheet_name="sheet", index=False)
-        check(assert_type(pd.read_excel(path), pd.DataFrame), pd.DataFrame)
-    with ensure_clean() as path:
-        df.to_excel(path, engine="openpyxl", header=["x", "y"])
-        check(assert_type(pd.read_excel(path), pd.DataFrame), pd.DataFrame)
-    with ensure_clean() as path:
-        df.to_excel(path, engine="openpyxl", columns=["col1"])
-        check(assert_type(pd.read_excel(path), pd.DataFrame), pd.DataFrame)
+    with pytest_warns_bounded(
+        DeprecationWarning,
+        match="datetime.datetime.utcnow",
+        lower="3.11.99",
+        version_str=platform.python_version(),
+    ):
```

Comment:
Given that this pattern is appearing all over the place, and hopefully we can one day remove it, it might be better to just create a special item in `tests/__init__.py` that returns  
```python
 pytest_warns_bounded(
        DeprecationWarning,
        match="datetime.datetime.utcnow",
        lower="3.11.99",
        version_str=platform.python_version(),
    )
```
Then we know there is one place that is catching the same warning.

Reply:
I found a nicer solution :) `pytest_warns_bounded` is used in only one place now, so I don't think creating a new function for it would make sense anymore.